### PR TITLE
Fix rust build script and contracts import path

### PR DESCRIPTION
**Pull request by nz-neooo**
Update rust build script use npm replace yarn, and panic when failure.
Update the some contracts import path.

### CODE REVIEW

File: contracts/PermissionedExchange.sol [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 pragma solidity ^0.8.15;
 
-import '../node_modules/@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol';
-import '../node_modules/@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol';
-import '../node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol';
-import '../node_modules/@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+import '@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol';
```
Comment (mzxyz) [inline contracts/PermissionedExchange.sol:6]: Jason mentioned that the reason for these import is for fixing issue of fuzzy test, would be good to confirm that

Comment (nz-neooo) [reply contracts/PermissionedExchange.sol:6]: Yeah, we need to confirm, let me have a test with fuzzy

Comment (Jianzhuo) [reply contracts/PermissionedExchange.sol:6]: Perhaps, we can make a branch for fuzzy test

Comment (mzxyz) [reply contracts/PermissionedExchange.sol:6]: T'his sounds good to me, will merge this PR first, @Jianzhuo  can you create a separate branch for fuzzy teste

**Pull request merged**